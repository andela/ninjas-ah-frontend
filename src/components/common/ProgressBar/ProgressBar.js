import React from 'react';
import './ProgressBar.scss';

const ProgressBar = () => (
  <div className="wrap-progress-bar">
    <span>{''}</span>
  </div>
);

export default ProgressBar;
