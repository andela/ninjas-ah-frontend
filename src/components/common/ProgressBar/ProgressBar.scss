@import '../../../assets/css/_colors';

.wrap-progress-bar {
  position: fixed;
  height: 6px;
  top: 0;
  left: 0;
  width: 100%;
  z-index: 99000000000;
  @extend .primary;
}
.wrap-progress-bar::before {
  content: '';
  position: absolute;
  width: 30%;
  height: 6px;
  top: 0;
  background: $success;
  animation: progress-bar 4s infinite;
}
@keyframes progress-bar {
  0% {
    left: -30%;
    background: $success;
  }
  50% {
    left: 100%;
    background: $danger;
  }
  100% {
    left: -30%;
    background: $black;
  }
}
