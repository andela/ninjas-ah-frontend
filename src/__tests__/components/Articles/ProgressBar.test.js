import React from 'react';
import { ProgressBar as ProgressBarComponent } from '../../../components/common/ProgressBar/ProgressBar';
import { shallow } from '../../../../config/enzymeConfig';

describe('<Progress />', () => {
  const component = shallow(<ProgressBarComponent />);
  it('should render progressbar', () => {
    expect(component).toHaveLength(1);
  });
});
