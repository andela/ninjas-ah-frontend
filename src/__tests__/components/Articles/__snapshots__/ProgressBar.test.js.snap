// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Progress /> should render progressbar 1`] = `ShallowWrapper {}`;
